### PR TITLE
cyl vs cart coord, 2D vs 3D

### PR DESCRIPTION
**Pull request by sagitta42**
Enabling the choice of cylindrical (default) and cartesian coordinates; 2D (phi symmetry, default) and 3D simulation (for cyl)

### CODE REVIEW

File: src/detector.jl [hunk 1]
```diff
@@ -144,18 +146,57 @@ function ssd_config(meta::PropDict, env::Environment)
...
+    # kind of initialize so that it exists out of the loop?    
+    grid = Dict()
+
+    if ssd_sim.coord == "cylindrical"
+        grid = Dict(
+            "axes" => Dict(
+                "r" => Dict("to" => cylinder_radius*1.2, "boundaries" => "inf"), # leave some margin
+                "phi" => Dict("from" => 0, "to" => ssd_sim.comp == "3D" ? 360 : 0, "boundaries" => "periodic"),
+                "z" => Dict(
+                    "from" => -10,
```
Comment (lmh91) [inline src/detector.jl]: I would suggest to use `-0.2*cylinder_height` here

**Review by lmh91 — CHANGES_REQUESTED** [review]
Very nice. Just a small thing. See my comment regarding `z` of the world.

File: src/detector.jl (hunk 1 above)
Comment (sagitta42) [reply src/detector.jl]: Done!

**Review by lmh91 — APPROVED** [review]
Great!

**Pull request merged**